CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# [4.11.0](https://github.com/algolia/instantsearch.js/compare/v4.10.0...v4.11.0) (2021-01-14)
+
+
+### Bug Fixes
+
+* **index:** do not warn for nested index widget ([#4620](https://github.com/algolia/instantsearch.js/issues/4620)) ([7502744](https://github.com/algolia/instantsearch.js/commit/7502744cd546181ec4429cd6b8144200ba2a8f82))
+* **insights:** don't quote values ([#4619](https://github.com/algolia/instantsearch.js/issues/4619)) ([ac2444c](https://github.com/algolia/instantsearch.js/commit/ac2444c36c6f41e35ed6d1a6d045479b35416576))
```

Comment:
```suggestion
* **index:** do not warn for nested index widget ([#4620](https://github.com/algolia/instantsearch.js/issues/4620)) ([7502744](https://github.com/algolia/instantsearch.js/commit/7502744cd546181ec4429cd6b8144200ba2a8f82))
* **insights:** don't quote values ([#4619](https://github.com/algolia/instantsearch.js/issues/4619)) ([ac2444c](https://github.com/algolia/instantsearch.js/commit/ac2444c36c6f41e35ed6d1a6d045479b35416576))
**escape-highlight:** fix circular dependency and move file to `/lib/utils/` folder ([#4618]((https://github.com/algolia/instantsearch.js/pull/4618))) ([08dd7ce](https://github.com/algolia/instantsearch.js/commit/08dd7ce6ba1ab6eafa5cda0495585973ac7d1fb9))
```

The naming of the PR might be confusing hence why it's not here, but since it was already `published` I thought it should be named `chore(xx)`

Reply:
chore PRs don't go inside the changelog @shortcuts :)

Reply:
Good to know!